### ISSUE TITLE
About training segment model

### ISSUE THREAD

**Issue by Kiris-Wu** [xinntao/BasicSR#14]
Could you post the training set for segment model? It seems apart from COCO and ADE, you have combined some addtional images. And also, could you provide training details like hyper-parameters? I see Xiaoxiao's code but it looks like based on caffe. Since we have seg_arch.py, I'm not sure whether it is possible to train the segment model in Pytorch? Thanks in advance!

**Comment by xinntao** [comment]
Hi, @Kiris-Wu 
The training set (*OutdoorSeg*) can be download from [Google Drive](https://drive.google.com/drive/folders/1LIb631GU3bOyQVTeuALesD8_eoApNniB?usp=sharing). (It is a small dataset, so pre-training is necessary.)

I trained the model using Caffe with Xiaoxiao's code and converted to a pytorch model for testing. 
A pre-trained segmentation model on COCO dataset was used and I attached the log file, you can find some basic settings in the log file.
[Outdoor_v1_8GPUs_bic.log](https://github.com/xinntao/BasicSR/files/2473102/Outdoor_v1_8GPUs_bic.log)

For segmentation in pytorch, I did not have a try. I just search it, maybe [semantic-segmentation-pytorch](https://github.com/CSAILVision/semantic-segmentation-pytorch) is a good choice. It uses Synchronized Batch Normalization, which is important and has many pre-trained models.

**Comment by Kiris-Wu** [comment]
Thanks for your detailed reply. Here is another confusion. Did you use the full datatset of COCO to do the pretrain or just part of them? Since I see in the log, it shows descriptions like "A total of 9900 images." As we all know, the full set for COCO is around 13GB in 2014.

**Comment by xinntao** [comment]
@Kiris-Wu 
the log file is for fine-tuning on the merged outdoor scenes (OutdoorSeg) dataset. 

I used the whole COCO dataset for pretraining.
